src/main/webapp/consentGroupDocuments/ConsentGroupDocuments.js

```diff
@@ -100,6 +100,17 @@ class ConsentGroupDocuments extends Component {
     return [this.props.serverURL, projectType, "show", projectKey,"?tab=review"].join("/");
   };
 
+  redirectToInfoLink = (projectKey) => {
+    let key = projectKey.split("-");
+    let projectType = '';
+    if (key.length === 3) {
+      projectType = key[1].toLowerCase();
```

Comment:
This is not always true. There are quite a few legacy projects that are `ORSP-X` (and many dev projects start with `DEV-`) and you won't get the right value here. Better to get the type from the full project object itself rather than trying to guess what it might be.

Reply:
You're right! either way I removed this part of the code since this wasn't necessary here. I will take into consideration this comment though.

Reply:
Looks like this is still an issue in this PR. Is there a commit missing?